### PR TITLE
Add mobile background and fix regular background

### PR DESCRIPTION
**Pull request by waalbert**
- Adds a mobile background for screen widths $\le$ 768px
- Fixes background gradient for screen widths $>$ 768px because for certain widths there would be a visible separation in the current background

### CODE REVIEW

File: apps/site/src/app/(home)/sections/Landing/Title.module.scss [hunk 1]
```diff
@@ -1,4 +1,11 @@
+@use "bootstrap-utils" as bootstrap;
+
 .title {
-	width: clamp(0px, 40vw, 40vw);
-	height: clamp(0px, 14.53vw, 14.53vw);
+	width: clamp(0px, 70vw, 70vw);
```
Comment (Bl20052005) [inline apps/site/src/app/(home)/sections/Landing/Title.module.scss:4]: I think this can be resolved to just max(0px, 70vw)

**Review by Bl20052005 — DISMISSED** [review]

File: apps/site/src/app/(home)/sections/Landing/Title.module.scss (hunk 1 above)
Comment (Bl20052005) [reply apps/site/src/app/(home)/sections/Landing/Title.module.scss:4]: however both work so nothing's actually wrong

Comment (waalbert) [reply apps/site/src/app/(home)/sections/Landing/Title.module.scss:4]: Oh wait clamp isn't needed here. It was from an old artifact, so I will remove.

**Review by Bl20052005 — APPROVED** [review]
Looks pretty good to me

**Pull request merged**